src/autocomplete/clickhouse/clickhouse-autocomplete.ts

```diff
@@ -4,7 +4,7 @@ import * as c3 from 'antlr4-c3';
 import {ColumnAliasSymbol, TableSymbol} from '../../lib/symbol-table.js';
 import {
     AutocompleteData,
-    AutocompleteParseResult,
+    ClickHouseAutocompleteResult,
```

Comment:
I've thought about the naming, and I think that `AutocompleteResult` is a good naming, because it can contain errors we've encountered during autocompletion, we don't need to separate errors from it, or to think about different naming. Our repository provides autocompletion, and our autocompletion reports errors we've encountered.

Also it's short, which is good!